kpet/run.py

```diff
@@ -13,9 +13,21 @@
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 """Module where the `run` command is implemented"""
 from __future__ import print_function
+import tempfile
+import shutil
 from xml.sax.saxutils import escape, quoteattr
 from kpet.exceptions import ActionNotFound
-from kpet import utils
+from kpet import utils, targeted
+
+
+def print_test_cases(patches, dbdir):
+    """Print test cases by querying layout according list of patch files"""
+    tmpdir = tempfile.mkdtemp(suffix='kpet')
+    patches = utils.patch2localfile(patches, tmpdir)
+    src_files = targeted.get_src_files(patches)
+    test_cases = targeted.get_test_cases(src_files, dbdir)
+    print('Test cases suggested "{}"'.format(', '.join(test_cases)))
+    shutil.rmtree(tmpdir)
```

Comment:
Please put this under a `finally` for a `try` block wrapping everything after the directory is created. This way we'll make sure we won't leave the directory behind.

Reply:
Good catch! :). Done